_content/monographs/wonhyo-selected-works_muller-park-vermeersch.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Wonhyo Selected Works"
+authors:
+  - "Wonhyo"
+editor: "A. Charles Muller"
+translator: "A. Charles Muller and Jin Y. Park and Sem Vermeersch"
+external_url: "https://web.archive.org/web/20250505111157if_/http://www.acmuller.net/kor-bud/01_wonhyo.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1mJ4-FZiH78vXxOYhN9M15N_jL-Vq6a4X/view"
+course: korean
+tags:
+  - east-asian-roots
+year: 2012
+month: jun
+olid: OL60565256M
+publisher: "Jogye Order of Korean Buddhism"
+address: "Republic of Korea"
+volume: 1
```

Comment:
Ah, yeah, a bit confusing. We always use `number` for a series (whether it's a series of talks or books or whatever)